pypeit/par/pypeitpar.py

```diff
@@ -2573,8 +2573,18 @@ def __init__(self, reference=None, method=None, echelle=None, ech_nspec_coeff=No
                                   'the determination of the wavelength solution (`i.e.`, not in '\
                                   'WaveTilts).'
 
+        defaults['fwhm_spat_order'] = 0
+        dtypes['fwhm_spat_order'] = int
+        descr['fwhm_spat_order'] = 'This parameter determines the spatial polynomial order to use in the ' \
+                                   '2D polynomial fit to the FWHM of the arc lines. See also, fwhm_spec_order.'
+
+        defaults['fwhm_spec_order'] = 1
+        dtypes['fwhm_spec_order'] = int
+        descr['fwhm_spec_order'] = 'This parameter determines the spectral polynomial order to use in the ' \
+                                   '2D polynomial fit to the FWHM of the arc lines. See also, fwhm_spat_order.'
```

Comment:
Is it worth combining these two into a single parameter that provides the order in both dimensions?

Reply:
I thought about this, but I think only the `spat_order` will ever be changed. I don't feel strongly, but keeping it separate makes the most sense to me, since that's what's done with `WaveTilts` and the `WavelengthSolutionPar` for echelle spectrographs.

Reply:
Sounds good to me!